spec/views/hq/users/_form.html.erb_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+describe "hq/users/_form.html.erb", type: :view do
+
+  before :each do
+    @user = FactoryGirl.build_stubbed(:user, user_name: 'Frederick & Bloggs', email: 'fred.bloggs@example.com')
```

Comment:
I would like to see this cleaned up since the values of `user_name` & `email` are not used explicitly in the test.